recipes/propy/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "propy3" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 93057b55b58d6f63ece6ee994fd01ca6c01a370ea45792558ccc048c4fb49520
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+    imports:
+        - propy
+
+about:
+  home: https://github.com/MartinThoma/propy3
+  license: "GNU General Public (GPL)"
+  license_family: LGPL
+  license_file: LICENSE
```

Comment:
I cannot find anything about a license in the upstream repo? I would suggest first adding a proper license there as conda-forge can only accept licensed software.

Reply:
The author did not have any license for this package

Reply:
@BastianZim The package that i'm trying to upload to conda (propy) has a BSD License in PyPi. Can I fill the License field in the .yaml file with the BSD? I've tryied the PR upstream but without success so far.

Reply:
Generally, the actual text is required so we'd probably need that. Since your PR is very recent I think we're fine with waiting some more days.
>has a BSD License in PyPi

From what I can see on [PyPI](https://pypi.org/project/propy3/) the license is _GNU General Public License_ so I've added a PR changing that to the correct one and also including the license in the source distribution.